docs/reporting/script-example.asciidoc

```diff
@@ -17,12 +18,13 @@ curl \
 ---------------------------------------------------------
 // CONSOLE
 
-<1> POST method is required.
+<1> `POST` method is required.
 <2> Provide user credentials for a user with permission to access Kibana and X-Pack reporting.
-<3> The `kbn-version` header is required for all POST requests to Kibana.
-<4> The POST URL. You can copy and paste the URL for any report from the Kibana UI.
+<3> The `kbn-version` header is required for all `POST` requests to Kibana.
+**The value must match the dotted-numeral version of the Kibana instance.**
```

Comment:
I really wanted the code block above to show the actual "dotted-numeral version". In text paragraphs, we can use `{version}`, but that does not work in pre-formatted text. I'm not sure what the best solution is.

Reply:
@tsullivan do you mean the minor version? If so, I would say:

`The value must match the minor version of the Kibana instance. For example, version 7.3.`

Reply:
@lcawl might have a suggestion on how to do this.

Reply:
I think you can try to do something like this:
https://raw.githubusercontent.com/elastic/elasticsearch/7.2/docs/reference/setup/install/targz.asciidoc
i.e. I think the magic part is subs="attributes"

Reply:
That worked!
![image](https://user-images.githubusercontent.com/908371/61491519-46227d80-a964-11e9-8ecf-e49dc71858c3.png)